README.md

```diff
@@ -1,6 +1,130 @@
+there is a risk that one team might consume more than its fair share of resources. 
+Resource quotas are a tool for administrators to manage and mitigate this concern.
+
+However, the native ResourceQuotas in Kubernetes are admission-based, which can cause the resource quota 
+admission controller to become a bottleneck for the API Server during evaluation and ResourceQuota status update calls. 
+Additionally, native ResourceQuotas are designed to handle predefined pod resources such as `requests.cpu`, 
+`limits.cpu`, `requests.memory`, `limits.memory`, and more. 
+This design is not easily extendable due to the potential burden on the API server, exacerbating the bottleneck.
+
+Native pod resources are often considered implementation details that can change across different development 
+stages or versions of Kubernetes Operators. For example, consider KubeVirt:
+
+KubeVirt is an operator that enables the deployment and management of virtual machines alongside containerized applications. 
+KubeVirt workloads run inside pods, presenting specific challenges with resource quotas:
+
+1. VM Live Migration: During VM live migration, both source and target pods are required.
+Having both pods simultaneously affects quotas, potentially leading to migration issues and upgrade failures.
+2. Billing Considerations: Service providers typically bill VMs similarly to platforms 
+like OpenStack, based on RAM size and CPU threads. However, KubeVirt's pods must account 
+for infrastructure overhead, which complicates resource quota management.
+
+
+As a result, deploying KubeVirt in a multi-tenant cluster is highly challenging. 
+It requires significant resource allocation to handle live migrations and manage VM infrastructure 
+overhead. This also couples users to the VM implementation, making them dependent on infrastructure 
+overhead that can change unexpectedly.
+
+
+## Overcoming Resource Quota Challenges with ApplicationAwareResourceQuota
+
+To address these challenges, the AAQ Operator use a different approach compared to the 
+native ResourceQuota. To avoid API server overhead, the AAQ Operator introduces an alternative 
+quota implementation called ApplicationAwareResourceQuota.
+The ApplicationAwareResourceQuota is API compatible with the native ResourceQuota, meaning it shares the same [specification and status definitions](https://github.com/kubevirt/application-aware-quota-api/blob/ac6757c09496bfab205d499a41daa6ed8815eab6/pkg/apis/core/v1alpha1/types.go#L46).
+Please see `manifests/examples/arq-example.yaml` for an example manifest.
+
+This new quota is controller-based rather than admission-based. Here’s how it works:
+
+1. Scheduling Gates: Scheduling gates are added to pods within namespaces where the Quotas are enforced using a MutatingWebhookConfiguration Object.
+2. Controller Evaluation: Once a pod is created, a controller evaluates whether there is sufficient room for the pod in the namespace.
```

Comment:
```suggestion
2. Controller Evaluation: Once a pod is created, a controller uses native ResourceQuotas to evaluates whether there is sufficient room for the pod in the namespace.
```

Reply:
the controller doesn't use the native resourceQuota to evaluates it it only refer to the resource boundaries defined in the ApplicationAwareResourceQuota and affected by pod creation, deletion, or 
changes—like `limits.memory`, `pods`, `requests.ephemeral-storage`, and others.

Reply:
I also added some information regarding the native Quota management